CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### 🛑 Breaking changes 🛑
 
+- "otelcol" part of the exposed metrics that is set as Prometheus namespace is replaced the same explicit metrics prefix (#5939)
```

Comment:
To be clear to most users this will not be a breaking change as the prometheus metric exposed will be the same. 

```suggestion
- Update `otelcol_` part of the exposed Prometheus metrics to be set via a metric prefix instead of a namespace (#5939)
```